visualization/plot_simulation.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
```

Comment:
plot_measurementdata and plot_simulationdata look really similar. could one improve code quality be merging both essentially?